subsys/net/lib/zperf/Kconfig

```diff
@@ -4,6 +4,7 @@
 menuconfig NET_ZPERF
 	bool "zperf shell utility"
 	depends on SHELL
+	select NET_CONTEXT_RCVTIMEO if NET_UDP
```

Comment:
I think it'd be better to use `if NET_NATIVE_UDP` so that the library can still be used with offloaded devices.

Reply:
done